assignments/002-unix-intro/unix-intro.sh

```diff
@@ -0,0 +1,708 @@
+####         named directories for sample `A05_S5`. Make a directory named `A05_S5`
+####         and copy all those files into that new directory. List the directory
+####         when you are done.
+
+ls A05_S5/
+
+-----
+
+DPCh_plate1_A05_S5.bam  DPCh_plate1_A05_S5.R1.fq.gz  DPCh_plate1_A05_S5.R2.fq.gz  DPCh_plate1_A05_S5.sam
+
+#### Ex. 28: remove the directory `A05_S5` and its contents
+
+rm -r A05_S5/
+
+#### Ex. 29: Use `ls` to try to list the file `it_aint_here`
+
+ls it_aint_here
+
+-----
+
+ls: cannot access it_aint_here: No such file or directory
+
+
+#### Ex. 30: Now, do the same thing but redirect _stderr_ into a file called 
+####         `my_bad.txt`, and then catenate the contents of that file, and then remove it
+
+
+ls it_aint_here 2> my_bad.txt
+cat my_bad.txt
+
+-----
+
+ls: cannot access it_aint_here: No such file or directory
+
+rm my_bad.txt
+
+
+#### Ex. 31: The files in `fastq` are text files, but they are all gzipped.
+####         Using `gzcat` and a pipe, view the first 8 lines of 
+####         `fastq/DPCh_plate1_C11_S35.R1.fq.gz`
+
+
+zcat fastq/DPCh_plate1_C11_S35.R1.fq.gz | head -n 8
+
+------
+
+@K00364:64:HTYYCBBXX:1:1101:1824:48192/1
+GTAGAATAATAGTGAATCAAATCAAATGTTATTTGTCACATGCGCTGAATACAACAGGTGTGGACCTTACAGTGAAATGCTTCCTTACAAGCCCTTAACCAACAATGCAGTTTTAAGAAAAATGAGTGTGAGATAAGTAAAAAATAGAAAA
++
+AAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJ7JJJJJJFFFFJJJJJJ7A7AJJAFAJJJJJFJJJJ-<JFJ-FFFJJJ
+@K00364:64:HTYYCBBXX:1:1101:2869:22080/1
+TTAAAACACGGTATGATGCAAGCAGCACAACACATCAATAACAAAAATACAAGAATTAGGGTCAGAAATCCAGTAACCACCATACTAGTGTACTTACCAAACCAGGCTCCCAACCAAGAGAACAGTCCAGACTCCTCCACCCTCGCCATGG
++
+-AAFFJJJJJJJJJJJJFFJJJFJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJ<JJJJJJFFFFJFFJJJJJJAFJJJJJJJJJJJJJJJJJJFAJJJJJJJJJJJJJJJJJJJFJJJFAJJJJJJJFJJJJJJJJFFJJ
+
+#### Ex. 32: Do the same to see the first 8 lines of `fastq/DPCh_plate1_C11_S35.R2.fq.gz`
+
+zcat fastq/DPCh_plate1_C11_S35.R2.fq.gz | head -n 8
+
+-----
+
+@K00364:64:HTYYCBBXX:1:1101:1824:48192/2
+CACAAGGAACGACAGAGGGTAGTGCGTACGGCCCAGCACATCACTGGGACCAAGCATCCTGCAATCCAGGACCTCCATACCAGGCGGTGTCAGAGGAAGGCCCTAAAATTGTCAAAGACTCCAGCCACCCTAGTCATTAACTCTTCGCTCT
++
+A<-A-<-F<AJ-<---<<--<-<-7FFJ-77A<-<--7-<7-<-7<J<77A<--7-AAA7-AF-7FAF<-<FJ---7-77FJA7-77F7FFJ7F<--<--77)A7FA-A7-<---7<<F7-<-7<))7--)<)7<AF----7<<<<-)-7)
+@K00364:64:HTYYCBBXX:1:1101:2869:22080/2
+CTATGATCAACAGCGTTTTGTGATTTACCCCCGTGATGCACTCACTGGTATGTCTGAACAGCTTGAGGCCACATCTAGGGTTGCCAGACAGAATAGACTTGCTTTGGATATGCTTCTTGCCAGTCAGGGGGGTGTCTGTAAGATGTTCGGT
++
+AAAFAFJJJJJJJJJJ-FJJJJJJFJJJJFJJJJJJJJJJJJJFJFFJFFFFFFJFFJJJFJFAAA777AAJJJAA--A<-A<-JJA-<JFJJJ-7<-AAJJFJJJ<JJ<JAFFFF---7<<7777<)7<<-<777-<-7--<FJJ<F-<<
+
+#text switched to green here, not sure why!
+
+#### Ex. 33: What do you notice about the names of the two reads in each 
+####         file (Lines 1 and 5, that start with a `@`)
+
+The names of the two reads are the same: K00364
```

Comment:
I think that K00364 might just be the run number.  The name of the read also includes all the rest:

```
@K00364:64:HTYYCBBXX:1:1101:2869:22080
```